### PR TITLE
Added support for multiple buckets

### PR DESCRIPTION
**Pull request by dancrumb**
I have an HTML web serving bucket and one for email templates.

I've updated the plugin to support that.

I'll annotate my changes in the PR so you can see why I made the changes (wherever they are not obvious).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,35 +3,5 @@ logs
 *.log
```
Comment (dancrumb) [inline .gitignore:3]: I just took out all the irrelevant ignores

File: index.js [hunk 2]
```diff
@@ -37,27 +38,39 @@ class Assets {
...
     const service = this.serverless.service;
-    const config = service.custom.assets;
+    let assetSets = service.custom.assets;
 
-    // glob
-    return new Promise((resolve, reject) => {
-      config.files.forEach((opt) => {
-        let cfg = Object.assign({}, globOpts, {cwd: opt.source});
-        glob.sync(opt.globs, cfg).forEach((fn) => {
+    if (!Array.isArray(assetSets)) {
```
Comment (dancrumb) [inline index.js:43]: All of this business is to ensure backwards compatibility for current users

**Comment by funkybob** [comment]
This looks great.  I think I originally planned this, but went with a MVP first :)

File: package.json [hunk 3]
```diff
@@ -4,12 +4,15 @@
...
+    "lint": "eslint index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Curtis Maloney",
+  "contributors": [
+    "Dan Rumney <dancrumb@gmail.com>"
+  ],
   "license": "MIT",
   "dependencies": {
-    "aws-sdk": "^2.7.10",
```
Comment (dancrumb) [inline package.json:12]: Looks like this isn't actually needed

File: README.md [hunk 4]
```diff
@@ -12,16 +12,22 @@ Add to your serverless.yml:
...
     assets:
-      bucket: my-bucket
-      files:
-        - source: ../assets/
-          globs: **/*.css
-        - source: ../app/
-          globs:
-            - **/*.js
-            - **/*.map
+       - bucket: my-bucket
```
Comment (funkybob) [inline README.md:15]: Shouldn't this same format change be made to the ACL example below?

Either that or comment that for a single bucket....

Comment (dancrumb) [reply README.md:15]: Yeah... strictly speaking, the ACL example would still work but I think I've just made it confusing. I'll go ahead and edit it.

File: package.json (hunk 3 above)
Comment (funkybob) [reply package.json:12]: Makes sense, it's implicitly required by Serverless, so long as you're using AWS already.

**Pull request merged**